packages/server/src/arpa_reporter/routes/users.js

```diff
@@ -20,7 +20,7 @@ async function validateUser (user, creator) {
     throw new Error('Role required')
   }
 
-  if (!agency_id) {
+  if (agency_id == null) {
```

Comment:
@disambiguator can we add some test coverage around this as well? I noticed late that there wasn't unit tests around this. Since this function us sed by many endpoints it should help improve overall repo coverage.